changelog.d/pr-7068.md

```diff
@@ -0,0 +1,6 @@
+### Internal
+
+- Ensure that `BatchedCommandError` is raised if the subprocesses of
+  `BatchedCommand` fails or raises a `CommandError`.  [PR
+  #7068](https://github.com/datalad/datalad/pull/7068) (by
+  [@christian-monch](https://github.com/christian-monch))
```

Comment:
@jwodder is reworking release helpers into a reusable action and it should get warping fixed, meanwhile

```suggestion
- Ensure that `BatchedCommandError` is raised if the subprocesses of `BatchedCommand` fails or raises a `CommandError`.  [PR #7068](https://github.com/datalad/datalad/pull/7068) (by [@christian-monch](https://github.com/christian-monch))
```